core-java/src/test/java/io/github/boykaframework/testng/api/restful/DataDrivenBookingTest.java

```diff
@@ -80,14 +80,13 @@ public void testBooking (final IDataRow row) {
 
     @Test (dataProviderClass = BookingDataProviders.class, dataProvider = "getBookingDataObject")
     public void testBookingObject (final BookingTestData bookingTestData) {
-        final var depositPaid = bookingTestData.getDepositPaid ()
-            .equalsIgnoreCase ("yes");
+        final var depositPaid = bookingTestData.getDepositPaid ();
```

Comment:
Is there any problem to use the previous change?

Reply:
Reverted to the old code.